recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/wayland-ivi-extension_1.11.0.bbappend

```diff
@@ -1,3 +1,3 @@
 SRCREV = "8fdae4cff2647d0060174509990cd2874e1d793d"
 BRANCH = "1.11"
-SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=http;branch=${BRANCH}"
+SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=http;branch=${BRANCH};protocol=https"
```

Comment:
Small comment here.
We have two `protocol` here/ Please remove first one `protocol=http;` so we will have same structure in all files: URL;branch;protocol.

With this small fixed, please merge this PR.